### DISCUSSION TITLE
Failed to parse OpenAPI file

### DISCUSSION THREAD

**Discussion by valpuia604** [dedoc/scramble#281] [Q&A]
Why I got this error in v0.8.5.
The error
```php
Failed to parse OpenAPI file
Please make sure your OpenAPI file is valid and try again
```
how to solve this?

**Comment by romalytvynenko** [comment]
Hey @valpuia604!

What exception do you see when you open `/docs/api.json` endpoint?

**Reply by Chris-Miracle** [reply]
TypeError
Dedoc\Scramble\Infer\Reflector\MethodReflector::getAstNode(): Return value must be of type PhpParser\Node\Stmt\ClassMethod, null returned

**Reply by valpuia604** [reply]
Hi @romalytvynenko sorry for late reply, I asked in Discord channel and already fixed. In my case my API function name and the function name inside controller mismatch (one has Capital letter, others don't)

**Reply by sangse0911** [reply]
@Chris-Miracle  i have same issue, do you fix this ?

**Reply by Chris-Miracle** [reply]
@sangse0911 nope, I have not been able to fix it, I just resolved to use postman

**Reply by yulchurin** [reply]
<img width="1504" alt="image" src="https://github.com/dedoc/scramble/assets/53298305/3ad489a5-a9ac-4459-ac78-1ea9c19fbc02">

open the /docs/api.json route, and then scroll down the screen
ignition will show you where problem is
